Cargo.toml

```diff
@@ -33,13 +33,15 @@ unindent = { version = "0.1.4", optional = true }
 [dev-dependencies]
 assert_approx_eq = "1.1.0"
 trybuild = "1.0.23"
-
-[build-dependencies]
-version_check = "0.9.1"
+rustversion = "1.0"
 
 [features]
 default = ["macros"]
 macros = ["ctor", "indoc", "inventory", "paste", "pyo3cls", "unindent"]
+# For CI
+all-stable = ["default", "num-bigint", "num-complex"]
```

Comment:
If this feature is just for convenience in CI scripts, maybe it's better to code it into the CI scripts instead of making this user-facing feature?

Reply:
:+1: